make/configure.py

```diff
@@ -1409,19 +1409,19 @@ def createCLI( cross = None ):
     h = IfHost( 'enable assembly code in non-contrib modules', 'NOMATCH*-*-darwin*', 'NOMATCH*-*-linux*', none=argparse.SUPPRESS ).value
     grp.add_argument( '--enable-asm', default=False, action='store_true', help=h )
 
-    h = IfHost( 'disable GTK GUI', '*-*-linux*', '*-*-*bsd*', none=argparse.SUPPRESS ).value
-    grp.add_argument( '--disable-gtk', default=False, action='store_true', help=h )
+    # GTK GUI is disabled by default on macOS and Windows, enabled otherwise
+    require_gtk = not (host_tuple.match('*-*-mingw*', '*-*-cygwin*', '*-*-darwin*'))
```

Comment:
I'm not sure I like inverting the logic here. Would rather allowlist BSD and Linux.

Reply:
I understand where you're coming from. I'll change it.